tools/cdk-release/lib/lifecycles/changelog.ts

```diff
@@ -50,9 +92,8 @@ export async function changelog(
   const presetConfig = await conventionalChangelogPresetLoader({
     name: 'conventional-changelog-conventionalcommits',
   });
-  debugObject(args, 'conventionalChangelogPresetLoader returned', presetConfig);
 
-  return new Promise((resolve, reject) => {
+  return new Promise<string>((resolve, reject) => {
```

Comment:
Do we need this type for some reason - TypeScript can't infer it...?

Reply:
Legacy holdouts from some other (more complex) types I was returning. Removed.